Remove MGM CPB (#1335)
corda/corda-docs-portal: content/en/platform/corda/5.0/application-networks/creating/mgm/cpi.md

   {{% /tab %}}
   {{< /tabs >}}

{{< note >}}

> Unlike other CPIs, an MGM CPI does not require a CPB file.

Reviewer: Technically no CPIs _require_ a CPB file so it's not specific to the MGM's CPI. It makes less sense that a non-MGM virtual node would install a CPI without a CPB, but it is possible.

Author: ```suggestion
An MGM CPI only requires a `GroupPolicy.json` file. A CPB file is not required. 
```

Reviewer: Looks great, thanks! 👍
